tests/integration/search.test.js

```diff
@@ -0,0 +1,11 @@
+describe('App test', () => {
+  it('loads successfully', () => {
+    cy.visit('/search');
+    cy.get('input[type="text"]').should('exist');
+  });
+  it('get results', () => {
+    cy.visit('/search');
+    cy.get('input[type="text"]').type('Health {enter}');
+    cy.get('.search-container').should('exist');
```

Comment:
I can't a test that the real results were gotten. Please add a comprehensive e2e test